src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -12,6 +12,7 @@ import {
 import {debounce} from 'lodash';
 import {useNavigation} from '@react-navigation/core';
 
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` is new in this hunk. Ensure that all the props used in this component are validated using PropTypes for better maintainability and to catch potential bugs during development.
```diff
+ import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->